src/components/ButtonPanel.js

```diff
@@ -1,50 +1,66 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-const renderButton = (value, bgColor, width) => (
-  <Button
-    value={value}
-    bgColor={bgColor}
-    width={width}
-  />
-);
-
-const ButtonPanel = () => (
-  <>
-    <div className="button-row">
-      {renderButton('AC')}
-      {renderButton('+/-')}
-      {renderButton('%')}
```

Comment:
- [ ] <b>REQUIRED:</b> Here, percentage sign stands for percentage not modulus. Kindly fix that.